config/routes.rb

```diff
@@ -2,14 +2,16 @@
 
 DockerManager::Engine.routes.draw do
   scope "/admin", constraints: AdminConstraint.new do
-    get "/upgrade" => "admin#index"
-    get "/upgrade/:id" => "admin#index"
+    get "/upgrade", to: redirect("/admin/update")
+    get "/upgrade/:id", to: redirect("/admin/update/%{id}")
+    get "/update" => "admin#index"
+    get "/update/:id" => "admin#index"
     get "/docker/repos" => "admin#repos"
     get "/docker/latest" => "admin#latest"
     get "/docker/progress" => "admin#progress"
     get "/docker/ps" => "admin#ps"
 
-    post "/docker/upgrade" => "admin#upgrade"
-    delete "/docker/upgrade" => "admin#reset_upgrade"
+    post "/docker/update" => "admin#update"
+    delete "/docker/update" => "admin#reset_update"
```

Comment:
I don't see updates to the controller actions, do we need to do that as well?

Reply:
Good catch! I now reverted the changes in route methods. I'm not going rename the method names in this PR (only the routes).